tests/integration.rs

```diff
@@ -6017,14 +6019,17 @@ mod parallel {
 
         let mut guest_command = GuestCommand::new(&guest);
         guest_command
-            .args(&["--cpus", "boot=1"])
+            .args(&["--cpus", "boot=2"])
             .args(&["--memory", "size=512M"])
             .args(&["--kernel", kernel_path.to_str().unwrap()])
             .args(&["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE])
             .default_disks()
             .args(&["--api-socket", &api_socket]);
 
-        let net_params = format!("fd={},mac={}", tap_fd, guest.network.guest_mac);
+        let net_params = format!(
+            "fd=[{},{}],mac={},num_queues=4",
```

Comment:
It's unfortunate we ended up with "fd=" in the command line and `fds` in the `NetConfig` struct.